modules/phase_field/src/materials/GBDependentTensorBase.C

```diff
@@ -0,0 +1,29 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#include "GBDependentTensorBase.h"
+
+template<>
+InputParameters validParams<GBDependentTensorBase>()
+{
+  InputParameters params = validParams<Material>();
+  params.addRequiredCoupledVar("gb", "GB variable");
+  params.addParam<Real>("bulk_parameter", 0.0, "Parameter value of bulk material");
+  params.addParam<Real>("gb_parameter", 0.0, "Parameter value at GB");
+  params.addParam<MaterialPropertyName>("gb_normal_tensor_name", "Name of GB normal tensor variable");
+  params.addParam<MaterialPropertyName>("gb_tensor_prop_name", "Name of GB tensor property");
```

Comment:
This is a little confusing, in the doc string you call it a variable once and a property in the next one even though both are MaterialPropertyNames. What is the difference?